apps/test/util/netsimTestUtils.js

```diff
@@ -158,7 +159,9 @@ exports.fakeShard = function () {
  */
 exports.initializeGlobalsToDefaultValues = function () {
   NetSimLogger.getSingleton().setVerbosity(NetSimLogger.LogLevel.NONE);
+  // Deep clone level so that changes we make to it for testing don't bleed
+  // into other tests.
   netsimGlobals.setRootControllers({}, {
-    level: levels.custom
+    level: _.clone(levels.custom, true)
```

Comment:
I was seeing changes to the level bleed over from one test to another.  This solves that issue, now we change the copy in the test and we make a new copy every time.